README.md

```diff
@@ -1963,7 +1990,7 @@ to taxes on equity compensation:
   and include a self-addressed, stamped envelope for the IRS to send you a return receipt.
   (Some people are so concerned about this they even ask a friend to go with them to the
   post office as a witness!)
-- ❗ Watch out for [the AMT trap](#the-amt-trap) we’ve already discussed.
+- ❗ Watch out for the AMT trap we’ve already discussed.
```

Comment:
These tax dangers can and should also be doled out to their relevant sections. See INTERNAL.